src/main/java/model/Walker.java

```diff
@@ -96,4 +188,14 @@ private void moveHorizontal() {
     public void switchDirection() {
         setFacingRight(!isFacingRight());
     }
+
+    private int randInt() {
+        Random rand = new Random();
+        int min = 1;
+        int max = 200;
+        int res = rand.nextInt((max - min) + 1) + min;
+        System.out.println(res);
```

Comment:
Please take out prints used for debugging. If you really need to display this information, use the Logger and a String that makes clear what this number means.